.travis.yml

```diff
@@ -1,3 +1,6 @@
+env:
+- API_REGEX='Google\.(Cloud\.)?[A-M].*'
```

Comment:
This one looks like it will capture all of them, which is probably why one of the splits is taking 2x as long. For example, in `Google.Cloud.Vision.V1`, the 'V' fails to be matched by `[A-M]`, so the optional `(Cloud\.)` cannot be used, so the regex will fall back to capturing the leading 'C' from "Cloud" with the `[A-M]` pattern and then `.*` will capture "loud.Vision.V1"